README.md

```diff
@@ -78,25 +91,42 @@ Then, make each route respond with a one-word string containing the RESTful acti
 var express = require("express");
 var app = express();
 
-// Your code starts here...
+app.get("/", function(req, res){
+  res.send("index");
+});
+
+app.post("/", function(req, res){
+  res.send("create");
+});
+app.post("/:id", function(req, res){
+  res.send("update");
+});
+
+app.delete("/:id", function(req, res){
+  res.send('delete');
+});
 
-```
 
-```js
-// Your answer...
 ```
+
+
 ### Question #7
 
 Describe the differences between Express and Rails as backend frameworks.
 
 ```text
-
+Rails used Active Record to give it CRUD functionality and PostgreSQL for creating relational databases. Express on the other hand uses Mongoose for CRUD functionality and MongoDB (non-relational database).
 ```
 
 ### Question #8
 
-What is the importance of using body-parser in our express application for post requests? Then, please provide an example of an Express App POST request: 
+What is the importance of using body-parser in our express application for post requests? Then, please provide an example of an Express App POST request:
 
 ```js
 
+// body-parser extracts the necessary data from incoming requests such as forms.
+app.post("/person"), function(req, res){
+  res.json(req.body);
+});
```

Comment:
:+1:  Kind of.  This is a debugging step, to ensure the route is getting the appropriate information from the form.  We would need to update this code and do something with `req.body`.